internal/describe/application.go

```diff
@@ -51,7 +51,16 @@ func (exe *applicationExecutor) init(controlPlane config.ControlPlane) (err erro
 	if err != nil {
 		return
 	}
-	exe.msvcs = msvcListResponse.Microservices
+
+	// Filter system microservices
+	for _, msvc := range msvcListResponse.Microservices {
+		// 4 is hard coded. TODO: Find a way to maintain this ID from Controller.
+		// Catalog item 1, 2, 3 are SYSTEM microservices, and are not inspectable by the user
+		if msvc.CatalogItemID != 0 && msvc.CatalogItemID < 4 {
```

Comment:
You have this magic number and explanatory comment in 4 places. Please move the number to a single place as a constant and reuse. Could make a function that returns bool based on this if statement

Reply:
Done